website/gridsome.config.js

```diff
@@ -30,13 +30,13 @@ module.exports = {
       {
         version: "v0.9",
         url: "/docs/v0.9/",
-        latest: false,
-        prerelease: true,
```

Comment:
redirects should be updated as well (I have a change on my computer with that :)